sem_covid/entrypoints/notebooks/pycaret_pwdb_all_classes.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<p>ouch! tta confussion matrix is confusing :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/meaningfy-ws/sem-covid/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>